app/tests/backend/lib/graphql/resolveFileUpload.test.ts

```diff
@@ -12,7 +16,7 @@ describe('The saveRemoteFile function', () => {
 
   it('should throw an error with unsupported payload', async () => {
     await expect(saveRemoteFile({})).rejects.toThrow(
```

Comment:
A bit late, but shouldn't this be: 
```suggestion
    await expect(saveRemoteFile(undefined)).rejects.toThrow(
```
?